app/service/rpc_service.ts

```diff
@@ -35,6 +35,21 @@ class RpcService {
     (window as any)._rpcService = this;
   }
 
+  debuggingEnabled(): boolean {
+    const url = new URL(window.location.href);
+    let sp = url.searchParams.get("debug");
+    if (sp == null) {
```

Comment:
```suggestion
    if (sp === null) {
```

Using strict equality checks to avoid unwanted type coercion